resources/views/cruds/datagrids/filters/_array.blade.php

```diff
@@ -1,28 +1,39 @@
+
+@php
+    $options = [
+        '' => __('crud.filters.options.include'),
+        'children' => __('crud.filters.options.children'),
+        'exclude' => __('crud.filters.options.exclude'),
+        'none' => __('crud.filters.options.none'),
+    ];
+    if (!isset($field['withChildren']) || $field['withChildren'] !== true) {
+        unset($options['children']);
+    }
+    if ($field['type'] == 'selectMultiple' && isset($models)) {
+        $selectedModels = $models;
+        $ids = array_keys($selectedModels);
+    } elseif (!empty($model) ) {
+        $selectedModels = [$model->id => $model->name];
+    } else {
+        $selectedModels = [];
+    }
+@endphp
 <div class="grid grid-cols-4 gap-2">
     <div class="col-span-3 text-left">
         <x-forms.select
-            :name="$field['field']"
-            :options="!empty($model) ? [$model->id => $model->name] : []"
-            :id="$field['field']"
+            :name="($field['type'] == 'selectMultiple') ? $field['field'] . '[]' : $field['field']"
+            :options="$selectedModels"
+            :selected="$ids ?? null"
             class="select2 entity-list-filter"
+            :multiple="$field['type'] == 'selectMultiple'"
             :extra="[
                 'data-url' => $field['route'],
                 'data-placeholder' => $field['placeholder'],
                 'data-dropdown-parent' => '#datagrid-filters'
             ]" />
     </div>
     <div class="col-span-1 field">
-        @php
```

Comment:
Just to understand, why was this removed? In some other filters like on the Creature page, we need to remove the children option on some of the filters.